build/crd-samples/devices/CC2650-device-instance.yaml

```diff
@@ -0,0 +1,37 @@
+  
+apiVersion: devices.kubeedge.io/v1alpha1
+kind: Device
+metadata:
+  name: sensor-tag-instance-01
+  labels:
+    description: TISimplelinkSensorTag
+    manufacturer: TexasInstruments
+    model: cc2650-sensortag
+spec:
+  deviceModelRef:
+    name: sensor-tag-model
+  protocol:
+    bluetooth:
+      macAddress: "BC:6A:29:AE:CC:96"   #pls note it is optional to provide MAC address
+  nodeSelector:
+    nodeSelectorTerms:
+    - matchExpressions:
+      - key: ''
+        operator: In
+        values:
+        - edge-node1          #pls give your edge node name
+status:
```

Comment:
logically speaking we could specify the desired state , but when we are creating a raw device instance for the first time , it is expected that the device will need to be provisioned , needs to connect to the cloud after it boots up and maybe it will report the actual state first , or it could be in an unknown state till then.
Then the user could play around with the desired state from the cloud.
also how will a user put the timestamp when putting desired value ?

Reply:
I guess that makes sense, i'll remove the desired section for both the properties under status from this device instance then, the user can add the desired status for the properties after the device gets connected